activity.py

```diff
@@ -1,9 +1,101 @@
-from olpcgames import activity
 from gettext import gettext as _
 
-class Activity(activity.PyGameActivity):
-    """Your Sugar activity"""
-    
-    game_name = 'menu:main'
-    game_title = _('falabracman')
-    game_size = None
+import sys
+import gi
+gi.require_version('Gtk', '3.0')
+from gi.repository import Gtk
+import pygame
+
+from sugar3.activity.activity import Activity
+from sugar3.graphics.toolbarbox import ToolbarBox
+from sugar3.activity.widgets import ActivityToolbarButton
+from sugar3.graphics.toolbutton import ToolButton
+from sugar3.activity.widgets import StopButton
+
+
+sys.path.append('..')  # Import sugargame package from top directory.
```

Comment:
Normally we have the sugargame folder as one of the folders in the activity, then it can be imported with 
```python 
from sugargame.canvas import PygameCanvas
```
OR
```python
import sugargame
```

Reply:
Ok thanks, I will add it and commit, maybe thats what i missed

Reply:
Fixed https://github.com/sugarlabs/falabracman-activity/pull/10/commits/98d7858ac5e6d23658bb4c509b2d3fdbcf87c686